functions/concepts-requests/main.py

```diff
@@ -0,0 +1,36 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START functions_concepts_requests]
+def make_request(request):
```

Comment:
This is the same sample as connection_pooling in the tips. Can we deduplicate?

Reply:
They're _slightly_ different - see [these two lines](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/8554/files#diff-70a8d629fb7746af08c0b8ff70969f0ae09b451862cf8cfb44e856c316a639f3R18-R19).

(That one uses connection pooling, while this one doesn't.)